component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -1557,4 +1569,26 @@ protected boolean retryAuthenticationEnabled() {
 
         return configProperties;
     }
-}
\ No newline at end of file
+
+
+    protected void triggerEvent(String userName, String tenantDomain, String userStoreDomainName,
```

Comment:
Add a method level comment and check this throughout the PR

Reply:
Added in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/26/commits/4c8a1a700c541009035ba290999b9609af6e95d2